x-pack/test/accessibility/apps/painless_lab.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['common', 'security']);
+  const testSubjects = getService('testSubjects');
+  const find = getService('find');
+  const a11y = getService('a11y');
+
+  describe('Accessibility Painless Lab Editor', () => {
+    before(async () => {
+      await PageObjects.common.navigateToApp('painlessLab');
+      await a11y.testAppSnapshot();
+      expect(await testSubjects.exists('painless_lab')).to.be(true);
```

Comment:
I'd pull this out and make it it's own test... Seems a little odd to me to `expect` within the `before` block.


```suggestion
    });
    
    it('loads default page with no a11y errors', async () => {
      expect(await testSubjects.exists('painless_lab')).to.be(true);
      await a11y.testAppSnapshot();
```

Reply:
yes sounds right..extracted it into its own test.